### ISSUE TITLE
Crash Error With Example Code

### ISSUE THREAD

**Issue by jeremykentbgross** [Dexus/pem#41]
Basic usage seems to crash.  I have a slightly more complex setup, but using the example code also gives me the same error (below).  I am running Win7 64bit if that helps.
# Example Code (from your page that gives same error)

``` javascript
var https = require('https'),
    pem = require('pem'),
    express = require('express');

pem.createCertificate({days:1, selfSigned:true}, function(err, keys){
  var app = express();

  app.get('/',  requireAuth, function(req, res){
    res.send("o hai!");
  });

  https.createServer({key: keys.serviceKey, cert: keys.certificate}, app).listen(443);
});
```
# My package.json:

```
{
    "name" : "Test",
    "description": "Test",
    "version" : "0.0.1",
    "author" : "Me <me@gmail.com>",
    "private" : true,
    "contributors" :
    [{
        "name" : "Me",
        "email" : "me@gmail.com"
    }],
    "dependencies" :
    {
        "express": "4.11.0",
        "node-inspector" : "0.8.3",
        "pem": "1.4.5"
    }
}
```
# My Code:

``` javascript
var express = require('express')
var https = require('https');
var pem = require('pem');

var app = express();

app.get(
    '/:id'
    ,function(inRequest, inResponse)
    {
        console.log(inRequest);
        inResponse.send('Hello World!');
    }
);

function ServerStarted()
{
    var aHost = server.address().address;
    var aPort = server.address().port;

    console.log('Listening at http://%s:%s\n', aHost, aPort);
}

console.log('test 1');

//Docs for creating better security feature set: https://github.com/andris9/pem
pem.createCertificate(
    {days:365, selfSigned:true}
    ,function(inError, inKeys)
    {
        console.log('test 2');
        if(inError)
        {
            console.error(inError);
            //system quit?
            return;
        }
        https.createServer(
            {key: inKeys.serviceKey, cert: inKeys.certificate}
            ,app
        ).listen(443, ServerStarted);
    }
);
```
# The Error:

```
v0.10.26
test 1

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:988:11)
    at Process.ChildProcess._handle.onexit (child_process.js:779:34)
```

**Comment by jeremykentbgross** [comment]
I discovered something interesting from reading some of the other bugs which might be of help.  One of the other issue reports referred to using 'which openssl'.

I can make this call ('which openssl') from my windows context menu>git bash and it works, but it does not from windows cmd.  So on a hunch I tried running node from my git bash instead of the usual bat script.  Then the PEM test app worked perfectly.

That will let me get by for the time being for testing my prototype, but ideally I need to launch this from a bat file in windows, and probably eventually from windows with no git/bash installed.  Is that (going to be) possible?
